include/llvm-dialects/Dialect/Visitor.h

```diff
@@ -369,28 +369,60 @@ class VisitorBuilder : private detail::VisitorBuilderBase {
 
   Visitor<PayloadT> build() { return VisitorBuilderBase::build(); }
 
+  //
+  // Add a simple visitor case for a single templatized OpDescription and a
+  // functor.
+  //
```

Comment:
Style nit: I don't think we're using empty comment lines like that anywhere else.